src/Libraries/Microsoft.Extensions.Compliance.Redaction/Microsoft.Extensions.Compliance.Redaction.csproj

```diff
@@ -20,6 +25,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Options" />
```

Comment:
This one is transitive to `Microsoft.Extensions.Options.ConfigurationExtensions`

Reply:
Removed.